modules/iceoryx/2.0.5/source.json

```diff
@@ -0,0 +1,9 @@
+{
+    "url": "https://github.com/eclipse-iceoryx/iceoryx/archive/refs/tags/v2.0.5.tar.gz",
```

Comment:
Does this repo offer release archives? Those are generally preferred as their hashes are guaranteed to remain stable over time, but it's also not a problem if they aren't available.

Reply:
Unfortunately, it doesn't offer release archives.